.github/workflows/wasmcloud.yml

```diff
@@ -216,9 +283,149 @@ jobs:
       - run: ${{ matrix.config.test-oci }}
         if: ${{ !endsWith(matrix.config.target, 'fhs') }}
 
-  build-windows:
+  build-provider-bin:
+    needs: [meta]
+    if: ${{ needs.meta.outputs.providers_modified == 'true' || startswith(github.ref, 'refs/tags/provider-') }}
+    strategy:
+      matrix:
+        config:
+          - name: blobstore-azure
+            target: aarch64-apple-darwin
+          - name: blobstore-azure
+            target: aarch64-unknown-linux-musl
+          - name: blobstore-azure
+            target: x86_64-apple-darwin
+          - name: blobstore-azure
+            target: x86_64-pc-windows-gnu
+          - name: blobstore-azure
+            target: x86_64-unknown-linux-musl
```

Comment:
Could this be simplified to a list of names and targets, e.g.
```yaml
config:
  name: [blobstore-azure, blobstore-fs, ...]
  target: [aarch64-apple-darwin, aarch64-unknown-linux-musl, ...]
```

I know that this wouldn't let us be flexible with, say, ignoring building just one provider for a specific target, but all of them look the same here to me